android/java/org/chromium/chrome/browser/preferences/BravePrefServiceBridge.java

```diff
@@ -171,6 +171,26 @@ public void setReferralDownloadId(String downloadId) {
         BravePrefServiceBridgeJni.get().setReferralDownloadId(downloadId);
     }
 
+    public void setP3AEnabled(boolean value) {
+        BravePrefServiceBridgeJni.get().setP3AEnabled(value);
+    }
+
+    public boolean getP3AEnabled() {
```

Comment:
@AlexeyBarabash can we use `GetBoolean` from src/components/prefs/android/pref_service_android.h. we can avoid below functons if it can help.

Reply:
@deeppandya 

The reason I made that is P3A settings are stored in local state prefs not the profile prefs, so maybe it wouldn't work. 
But I cannot find now where `PrefServiceAndroid` is initialized and which `PrefService` does it use (profile or local state), so I will  double check and leave the comment here.

Reply:
@deeppandya 
I gave up to find this in code, so checked by tracing pointers, yes, I cannot use `PrefServiceAndroid` for local_state, because they are using different `PrefService*`:
```
11-12 11:52:55.656: E/chromium(21494): [ERROR:chrome_pref_service_factory.cc(372)] [BraveSync] CreateLocalState 000 pref_filename=/data/user/0/com.brave.browser_default/app_chrome/Local State
11-12 11:52:55.690: E/chromium(21494): [ERROR:chrome_pref_service_factory.cc(373)] [BraveSync] CreateLocalState stack=#00 pc 0x000ea951 /data/app/com.brave.browser_default-RnSs4hV-gJYlPbam2dHjEQ==/lib/arm/libbase.cr.so
11-12 11:52:55.691: E/chromium(21494): [ERROR:pref_service.cc(105)] [BraveSync] PrefService 000 this=0xd6e2af80
11-12 11:52:56.388: E/chromium(21494): [ERROR:pref_service_android.cc(17)] [BraveSync] PrefServiceAndroid 000 pref_service=0xd6c05580

```